src/no-util.ts

```diff
@@ -0,0 +1,32 @@
+// no-util.ts
+
+/*
+ * Copyright (c) 2021-2024 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import type { Rule } from 'eslint';
+export default {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Detects if file name is util',
+      url: 'https://github.com/checkdigit/eslint-plugin',
+    },
+  },
+  create(context) {
+    return {
+      Program() {
+        const filename = context.filename;
+        const utilRegex = /util/iu;
```

Comment:
I think we need this regex to be more sophisticated, e.g. `electric-utility.ts` shouldn't be banned, but something like `electric-util.ts` should be.